wire-ios/Wire-iOS Tests/SwiftSnapshotTesting/SnapshotHelper+UIKit.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SnapshotTesting
+import UIKit
+import XCTest
+
+extension SnapshotHelper {
```

Comment:
These are additional verify methods as an extension but they don't seem to use the properties of the helper (except the precision). I think that could be confusing to the user of the helper. Could we unify the verify methods to behave consistently?

Reply:
Fixed it here: https://github.com/wireapp/wire-ios/commit/b569bd280263c125105f73db09591b3d71bdf590